### PR TITLE
[release-v1.7] SC settings for ApiServerSource's Receive Adapter's container/deployment

### PR DESCRIPTION
**Pull request by matzew**
SC settings for ApiServerSource's Receive Adapter's container/deployment

### CODE REVIEW

File: pkg/reconciler/apiserversource/apiserversource.go [hunk 1]
```diff
@@ -264,3 +273,67 @@ func (r *Reconciler) createCloudEventAttributes(src *v1.ApiServerSource) ([]duck
...
+	v, err := versioner.ServerVersion()
+	if err != nil {
+		return err
+	}
+	currentVersion, err := semver.Make(normalizeVersion(v.GitVersion))
+	if err != nil {
+		return err
+	}
+
+	minimumVersion, err := semver.Make(normalizeVersion("v1.25.0"))
```
Comment (matzew) [inline pkg/reconciler/apiserversource/apiserversource.go:289]: For 4.12 and plus?

**Review by creydr — APPROVED** [review]
/lgtm

**Pull request merged**